tests/sentry/api/endpoints/test_sentry_app_publish_request.py

```diff
@@ -53,7 +53,7 @@ def test_publish_already_published(self, send_mail):
         response = self.client.post(self.url, format="json")
         assert response.status_code == 400
         assert response.data["detail"] == "Cannot publish already published integration"
-        send_mail.asssert_not_called()
```

Comment:
o.o why did tests still pass?

Reply:
`send_mail` is a mock, so it'll let you call anything you want on it :(

Reply:
I attempted to change this to assert_not_called but sentry-flake8 gets angry at me. Then more weird stuff happened when I tried `assert send_mail.called`. \shrug

Reply:
``assert not send_mail.mock_calls``